doc/api/cli.md

```diff
@@ -288,6 +288,17 @@ Path to the file used to store the persistent REPL history. The default path is
 to an empty string (`""` or `" "`) disables persistent REPL history.
 
 
+### `NODE_TTY_UNSAFE_ASYNC=1`
+<!-- YAML
+added: REPLACEME
+-->
+
+When set to `1` writes to stdout and stderr will be non-blocking and async when
+outputting to a TTY on platforms which support async stdio.
```

Comment:
stdio is not a code thing, so I'm not going to backtick it.

Also people really should not use this anyways, so I'm not sure it matters if it is advanced wording.